File: PYTHON/Course.py
class Course:
    # constructor
    def __init__(self, nama_matakuliah):
        self._nama_matakuliah = nama_matakuliah

    # getter dan setter
    def getNamaMatakuliah(self):
        return self._nama_matakuliah

    def setNamaMatakuliah(self, nama_matakuliah):
        self._nama_matakuliah = nama_matakuliah

File: C++/Course.cpp
#include <iostream>
#include <string>
using namespace std;
#include "Course.h"

// constructor
Course::Course(string nama_matakuliah) {
    _nama_matakuliah = nama_matakuliah;
}

// getter dan setter
string Course::getNamaMatakuliah() const{
    return _nama_matakuliah;
}

void Course::setNamaMatakuliah(string nama_matakuliah) {
    _nama_matakuliah = nama_matakuliah;
}
